src/benchmarklib/synthetic_table_generator.cpp

```diff
@@ -50,28 +50,22 @@ namespace opossum {
 std::shared_ptr<Table> SyntheticTableGenerator::generate_table(const size_t num_columns, const size_t num_rows,
                                                                const ChunkOffset chunk_size,
                                                                const SegmentEncodingSpec segment_encoding_spec) {
-  auto table =
-      generate_table({num_columns, {ColumnDataDistribution::make_uniform_config(0.0, _max_different_value)}},
-                     {num_columns, {DataType::Int}}, num_rows, chunk_size,
-                     std::vector<SegmentEncodingSpec>(num_columns, segment_encoding_spec), std::nullopt, UseMvcc::No);
+  ColumnSpecification column_specification = {{ColumnDataDistribution::make_uniform_config(0.0, _max_different_value)},
+                                              DataType::Int,
+                                              std::nullopt,
+                                              std::nullopt};
+  auto table = generate_table({num_columns, column_specification}, num_rows, chunk_size,
+                              std::vector<SegmentEncodingSpec>(num_columns, segment_encoding_spec), UseMvcc::No);
 
   return table;
 }
 
 std::shared_ptr<Table> SyntheticTableGenerator::generate_table(
-    const std::vector<ColumnDataDistribution>& column_data_distributions,
-    const std::vector<DataType>& column_data_types, const size_t num_rows, const ChunkOffset chunk_size,
-    const std::optional<ChunkEncodingSpec>& segment_encoding_specs,
-    const std::optional<std::vector<std::string>>& column_names, const UseMvcc use_mvcc) {
+    const std::vector<ColumnSpecification>& column_specifications, const size_t num_rows, const ChunkOffset chunk_size,
+    const std::optional<ChunkEncodingSpec>& segment_encoding_specs, const UseMvcc use_mvcc) {
   Assert(chunk_size != 0ul, "cannot generate table with chunk size 0");
```

Comment:
Not your code, but please first letter in upper case and add a trailing dot.

Reply:
Fixed in ba0d0c488f478054b6a1606a86dcd6e23d775dd5.